.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "hpc-stack-mod"]
```

Comment:
Why does this need to be removed? The docs about HPC-Stack installation are being managed on HPC-Stack so that they can be maintained in one place rather than separately in every UFS app. I was under the impression that the submodule needed to be included in .gitmodules in order to pull their docs in. Is that incorrect?/Is there another way to pull in a submodule without including a .gitmodules file?

Reply:
Sorry, @gspetro!  I misunderstood how this file was being used in this case.  If it's to pull in additional documentation, then that's fine.